stock_quantity_history_location/tests/test_stock_quantity_history_location.py

```diff
@@ -26,30 +29,14 @@ def setUpClass(cls):
                 "company_id": cls.main_company.id,
             }
         )
-        # Create a move for the past
-        move = cls.env["stock.move"].create(
-            {
-                "name": "Stock move in",
-                "location_id": cls.supplier_location.id,
-                "location_dest_id": cls.child_test_stock_loc.id,
-                "product_id": cls.product.id,
-                "product_uom": cls.product.uom_id.id,
-                "product_uom_qty": 100.0,
-            }
-        )
-        move._action_confirm()
-        move._action_assign()
-        move_line = move.move_line_ids[0]
-        move_line.qty_done = 100.0
-        move._action_done()
-        move.date = "2019-08-11"
+        cls._create_stock_move(cls, location_dest_id=cls.child_test_stock_loc, qty=100)
 
     def test_wizard_past_date(self):
         wizard = self.env["stock.quantity.history"].create(
             {
                 "location_id": self.test_stock_loc.id,
                 "include_child_locations": True,
-                "inventory_datetime": "2019-08-12 00:00:00",
+                "inventory_datetime": fields.Datetime.now(),
```

Comment:
Why are you putting again dates as dynamic? Please let it fixed.

Reply:
These change are necessary according to re-define code in `_create_stock_move` function and removed date in `stock.move`.
It's not correct to set date because when done move, will be auto-create `stock.valuation.layer` record, and wizard process use these code: https://github.com/odoo/odoo/blob/13.0/addons/stock_account/models/product.py#L108-L110
Too, other addon (`stock_account_quantity_history_location`) depends to these test and it's necessary to work fine.

Reply:
And aren't you able to pass such date in the context?

Reply:
IMO is not possible because `open_at_date` function https://github.com/odoo/odoo/blob/13.0/addons/stock/wizard/stock_quantity_history.py#L35) use `inventory_datetime` field.

Reply:
And why aren't you forcing such date there? I'm trying to avoid random fails due to dates (or the timezone effect, where a test made around midnight fails).

Reply:
Because `create_date`  field is auto-fill but if you prefer, I revert this change and in another PR (https://github.com/OCA/stock-logistics-reporting/pull/146) it will be necessary to do it to prevent the tests from failing.